### PR TITLE
Support Markdown-rendered footer, update defaults

### LINKED ISSUE #2: Request: support Markdown in copyright and update label
It would be nice if the plugin surrounded the copyright and update label in a [Markdown block](https://toolchain.gitbook.com/templating/builtin.html) for the purpose of adding bold, links, etc.

### PR DESCRIPTION
**Pull request by panathea**
This PR:

- Adds Markdown support, fix #2
- Updates the readme
  - Bilingual
  - Cleaned up to be shorter and more concise
- Updates default values
  - for Gitbook → © All Rights Reserved
  - update → updated
- Clean up code
  - Defaults do not need to be handled manually, Gitbook handles them already
  - Hook is now async since gitbook's Markdown renderer is async

Please let me know if you have any questions or improvements!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,90 @@
...
+      "copyright": {
+        "type": "string",
+        "default": "© All Rights Reserved",
+        "title": "你的版权信息",
+        "description": "Copyright text"
+      },
+      "markdown": {
+        "type": "boolean",
+        "default": false,
+        "title": "",
```
Comment (panathea) [inline README.md:60]: Could use Chinese string for this property

File: assets/lib/plugin.js [hunk 2]
```diff
@@ -1,43 +1,34 @@
...
+    return Promise.all([wrapIfMarkdown(config.copyright), wrapIfMarkdown(config.update_label)])
+        .then(function(labels) {
+            var copyright = labels[0];
+            var updateLabel = labels[1];
+            page.content += [
+                '<footer class="page-footer-ex">',
+                    '<div class="page-footer-ex-copyright">',
+                        copyright,
+                    '</div>',
+                    '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;',
```
Comment (panathea) [inline assets/lib/plugin.js:25]: I need to test if adding a right-margin to the first div will work in PDF…that is why these nonbreaking spaces were added, right?

**Comment by zq99299** [comment]
Well, yes, they said pdf can read another set of configurations, but I do not know how to set up, pdf does not read css, so add a number of spaces

**Comment by zq99299** [comment]
On the pdf generated, the last time I tested with svg, but in the gitbook website books pdf, build failed, do not know what you did not solution

**Pull request merged**